server/views/index.pug

```diff
@@ -6,316 +6,217 @@ html
     title="Ch-Cat.ru"
 
     link(rel='stylesheet', href='/css/main-styles.css')
-    link(rel='stylesheet', href='/bootstrap/dist/css/bootstrap.min.css')
+    link(rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css" integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous")
```

Comment:
Good idea to use the cdn instead! Then hopefully no maintenance if something in the library changes 👍 😄